Corona-Warn-App/src/main/res/layout/fragment_submission_dispatcher.xml

```diff
@@ -141,7 +141,7 @@
                     android:layout_height="wrap_content"
                     android:layout_marginTop="@dimen/spacing_medium"
                     android:text="@string/submission_dispatcher_already_positive_subheadline"
-                    android:visibility="visible"
```

Comment:
Why was this visibility attribute present? Is the visibility toggled from code? If not, hiding here is good I think

Reply:
When I added the changes to the dispatcher screen #5689, I already made these tiles gone, but then @mtwalli asked me to add them back because we didn't have the decision to remove them final at the time. So I just used the visiblity attribute here to set them to visible for the time being.